lL2I_R: [-0.516377, -0.702254, -0.490096, 0.491997, -0.711704, 0.501414, -0.700923, 0.0177927, 0.713015]
lL2I_t: [-0.334623, 0.431973, 1.94043]

rL2I_R: [-0.514521, 0.701075, -0.493723, -0.492472, -0.712956, -0.499164, -0.701954, -0.0136853, 0.712091]
rL2I_t: [-0.333596, -0.373928, 1.94377]

# lL_topic: "/ns1/velodyne_points"
# rL_topic: "/ns2/velodyne_points"

rL_topic: "/ns1/velodyne_points"
lL_topic: "/ns2/velodyne_points"

L_topic: "/points_raw"


save_bag: true
bag_path: "/media/liu/PATHF/KAIST_DataSet/urban26/merged_lidar.bag"


